tests/golden/vault/common.bats

```diff
@@ -14,11 +14,11 @@ load '../helpers'
   assert_failure
   assert_output --partial - <<EOF
 Invalid path segment for target backend:
+Got: 'bad
+path'.
```

Comment:
Oh, we want to escape characters in invalid paths (e.g., we should show `\n` instead of ` `).

The reason for this is that some invalid characters might be hard to spot _unless_ they're escaped.

For example, at the moment, if I use a path with a ["zero-width space"](https://unicode-table.com/en/200B/), it won't be visible in the error message:

```
$ coffer create "$(echo "bad\xE2\x80\x8Bpath")"
Invalid path segment for target backend:
Got: 'badpath'.
      ^^^^^^^ seems like a valid path, but it's not, there's a zero-width space character in the middle.
```


If we escape all characters, then we would see:

```
$ coffer create "$(echo "bad\xE2\x80\x8Bpath")"
Invalid path segment for target backend:
Got: "bad\8203path".
```

To escape the characters, we can use `#s` instead of `#` to tell the `nyan-interpolation` to use the `Show Text` instance instead of the `Buildable Text` instance:

```hs
-- before
Got: '#{segment}'.

-- after
Got: #s{unPathSegment segment}.
```

Reply:
oh, that's nice)
done